tests/scripts/thread-cert/Cert_7_1_04_BorderRouterAsRouter.py

```diff
@@ -108,6 +116,53 @@ def test(self):
             if addr[0:10] == '2001:2:0:1' or addr[0:10] == '2001:2:0:2':
                 self.assertTrue(self.nodes[LEADER].ping(addr))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        ROUTER = pv.vars['ROUTER']
+        MED = pv.vars['MED']
+        SED = pv.vars['SED']
+        _rpkts = pkts.filter_wpan_src64(ROUTER)
+
+        # Step 3: The DUT MUST send properly formatted MLE Advertisements
+        # The DUT MUST send a CoAP Server Data Notification message
+        # with the server’s information (Prefix, Border Router) to the Leader
+        _rpkts.filter_mle_cmd(MLE_CHILD_ID_REQUEST).must_next()
+        _rpkts.filter_mle_cmd(MLE_ADVERTISEMENT).must_next().must_verify(
+            lambda p: {SOURCE_ADDRESS_TLV, LEADER_DATA_TLV, ROUTE64_TLV} == set(p.mle.tlv.type))
+        _pkt = _rpkts.filter_coap_request(SVR_DATA_URI).must_next()
+        _pkt.must_verify(lambda p: p.wpan.dst16 == pv.vars['LEADER_RLOC16'] and {
+            Ipv6Addr('2001:2:0:1::'), Ipv6Addr('2001:2:0:2::')
+        } == set(p.thread_nwd.tlv.prefix) and p.thread_nwd.tlv.border_router.flag.p == [1, 1] and p.thread_nwd.tlv.
+                         border_router.flag.s == [1, 1] and p.thread_nwd.tlv.border_router.flag.r == [1, 1] and p.
+                         thread_nwd.tlv.border_router.flag.o == [1, 1] and p.thread_nwd.tlv.stable == [1, 1, 0, 0])
+        _rpkts_med = _rpkts.copy()
+        _rpkts_sed = _rpkts.copy()
+
+        # Step 4: Automatically transmits a 2.04 Changed CoAP response to the DUT
```

Comment:
should we verify step 4?

Reply:
no need to check step 4 according to the test plan